portfolio/src/main/webapp/portfolio.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="My Portfolio">
+    <title>Keiffer Acoba | Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+
+    <!-- Navbar -->
+    <nav id="navbar">
+      <ul>
+        <li><a href="index.html">About</a></li>
+        <li><a href="#" id="here">Portfolio</a></li>
+        <li><a href="resume.pdf" target="_blank">Résumé</a></li>
+      </ul>
+    </nav>
```

Comment:
This is totally fine. However, there are some interesting ways to share this code across all pages of your site, minimizing the need to update the navbar in several places when any 1 place is updated. These seem to be accomplishable via ES6 modules, though I am not familiar with the technology. Something cool to think about, though!

Reply:
Oh nice, I didn't know there was a way to share code like across different html pages, that would definitely help with modularity. I'll look into it, thanks!